<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ITSClass</class>
 <widget class="QMainWindow" name="ITSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ITS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>700</width>
      <height>400</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::SubWindowView</enum>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>170</width>
      <height>510</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Munu</string>
     </property>
    </column>
   </widget>
   <widget class="QMdiArea" name="mdiDownSapmle">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>700</width>
      <height>110</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport_Photos">
     <property name="title">
      <string>Import Photos</string>
     </property>
     <addaction name="actionKlasor"/>
     <addaction name="actionSearch_HardDisk"/>
    </widget>
    <addaction name="menuImport_Photos"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionKlasor">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="actionWeb">
   <property name="text">
    <string>Web</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExport_Photos">
   <property name="text">
    <string>Export Photos</string>
   </property>
  </action>
  <action name="actionSearch_HardDisk">
   <property name="text">
    <string>Search HardDisk</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="its.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>ITSClass</receiver>
   <slot>treeWidgetSelectionChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mdiArea</sender>
   <signal>subWindowActivated(QMdiSubWindow*)</signal>
   <receiver>ITSClass</receiver>
   <slot>updateFace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>treeWidgetSelectionChange()</slot>
  <slot>updateFace()</slot>
 </slots>
</ui>
